applications/OptimizationApplication/python_scripts/responses/linear_strain_energy_response_function.py

```diff
@@ -0,0 +1,47 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.OptimizationApplication as KratosOA
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_info import OptimizationInfo
+from KratosMultiphysics.OptimizationApplication.execution_policies.execution_policy_decorator import ExecutionPolicyDecorator
+from KratosMultiphysics.OptimizationApplication.responses.response_function import ResponseFunction
+from KratosMultiphysics.OptimizationApplication.responses.response_function import SupportedSensitivityFieldVariableTypes
+
```

Comment:
can's we simply import LinearStrainEnergyResponseUtils without using KratosOA.ResponseUtils to save repeating things and space ?!

Reply:
I don't think we can. It's a limitation by python side.